code/game/machinery/bots/medbot.dm

```diff
@@ -46,10 +46,10 @@
 	name = "\improper Mysterious Medibot"
 	desc = "International Medibot of mystery."
 	skin = "bezerk"
-	treatment_oxy = "omnizine"
-	treatment_brute = "omnizine"
-	treatment_fire = "omnizine"
-	treatment_tox = "omnizine"
+	treatment_oxy = "tricordrazine"
+	treatment_brute = "tricordrazine"
+	treatment_fire = "tricordrazine"
+	treatment_tox = "tricordrazine"

```

Comment:
Omnizine is PAINFULLY slow. While it heals lots of damage types, it takes forever to do, and Mysterious medibots should be more powerful and faster than normal ones.